docs/source/interns/current-tasks.md

```diff
@@ -0,0 +1,53 @@
+(current-microtasks)=
+
+# List of Current Microtasks Available
+
+% NOTE: The CSV files used by this page are generated by helper-scripts/get-repo-issues.py
+
+This table shows the microtasks currently available to work on during a
+contribution period. The table contains a link to the issue describing the
+task, a label indicating which project it is connected to, an indication of
+how many applicants can work on the issue simultaneously, and a label indicating
+the difficulty level of the task.
+
+```{attention}
+If the table doesn't show, that means we don't currently have any open issues!
+```
+
+<div class="full-width hubs-table">
```

Comment:
Is the 'hubs-table' class here and on L36 meaningful, or leftover from a copy/paste?

Reply:
Copy-paste from the 2i2c docs! I will push a fix